hw_interface_plugin: "mrs_uav_fcu_api/Api"

comms_timeout: 1.0 # [s]





simulated_rtk:
  utm_zone: "32T"
  utm_x: 465710.758973
  utm_y: 5249465.43086
  amsl: 340.0

inputs:
  control_group: false
  attitude_rate: true
  attitude:      true

outputs:
  distance_sensor:      false
  gnss:                 true
  gnss_status:          true
  rtk:                  false
  imu:                  true
  altitude:             true
  magnetometer_heading: true
  magnetic_field:       true
  rc_channels:          true
  battery_state:        false
  position:             true
  orientation:          true
  velocity:             true
  angular_velocity:     true
  odometry:             false
  ground_truth:         false